### PR TITLE
Add section on multi-line imports. Fixes #1051

### LINKED ISSUE #1051: Best Practices for Multi-Line Imports?
It would be great if the style guide could weigh in on import lines that exceed the character limit (and thus need to be broken up on to multiple lines) ...

     import {longNameA, longNameB, longNameC, longNameD, longNameE} from '/some/pretty/long/path/

vs.

     import {longNameA, longNameB, longNameC, longNameD, longNameE}
     from '/some/pretty/long/path/

vs.

     import {longNameA, longNameB, longNameC, longNameD, 
             longNameE} from '/some/pretty/long/path/

vs. 

     import {longNameA, longNameB, longNameC, longNameD, 
             longNameE
    } from '/some/pretty/long/path/

vs. 

     import {
          longNameA,
          longNameB, 
          longNameC,
          longNameD,
          longNameE
      } from '/some/pretty/long/path/

vs. 

     import {
          longNameA,
          longNameB, 
          longNameC,
          longNameD,
          longNameE
      } 
      from '/some/pretty/long/path/

### PR DESCRIPTION
**Pull request by karanjthakkar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1223,6 +1223,25 @@ Other Style Guides
     foo.init();
     ```
 
+  <a name="modules--multiline-imports-over-newlines"></a><a name="10.8"></a>
```
Comment (ljharb) [inline README.md]: only legacy sections get a numbered link - please remove the 10.8 one here.

File: README.md [hunk 2]
```diff
@@ -1223,6 +1223,25 @@ Other Style Guides
...
+  - [10.8](#modules--multiline-imports-over-newlines) Multi-line imports should be preferred over single line
```
Comment (ljharb) [inline README.md]: this isn't accurate. It's totally fine to use multiple imports on a single line - the section we're looking for is simply to say that *if* the import is across more than one line, how it should be indented.

File: README.md [hunk 3]
```diff
@@ -1223,6 +1223,25 @@ Other Style Guides
...
+    // bad
+    import {longNameA, longNameB, longNameC, longNameD, longNameE} from 'path';
+
+    // good
+    import {
+      longNameA,
+      longNameB,
+      longNameC,
+      longNameD,
+      longNameE
```
Comment (ljharb) [inline README.md]: this line is missing a trailing comma (covered by the `comma-dangle` rule)

File: README.md (hunk 1 above)
Comment (karanjthakkar) [inline README.md]: Noted.

File: README.md (hunk 3 above)
Comment (karanjthakkar) [inline README.md]: Noted.

File: README.md (hunk 2 above)
Comment (karanjthakkar) [inline README.md]: How do you think the text should be framed then? I'm not able to figure out the right words for this.

Comment (karanjthakkar) [inline README.md]: Done.

**Comment by karanjthakkar** [comment]
@ljharb Have incorporated the feedback :)

**Pull request merged**